### DISCUSSION TITLE
June 12, 2024

### DISCUSSION THREAD

**Discussion by tcmitchell** [SynBioDex/SBOL-Editors#95] [Steering Committee Meetings]
# Participants

* Prashant Vaidyanathan
* David Ross
* Chris Myers
* Tom Gorochowski
* Yan Kay
* Goksel Misirli
* Tom Mitchell
* Gonzalo Vidal
* Lukas Buecherl
* Georgie Hau Sørensen

# Agenda

* Editor election results
* Keeping retired editors involved
* IWBDA updates
* COMBINE
* SBOL talk at NIST
    * Status, etc.
* New ISO standard
    * Might be relevant to SBOL adoption

# Discussion

* Chris volunteers to give a talk at NIST
* ISO standard: https://www.iso.org/obp/ui/en/#iso:std:iso:20691:ed-1:v1:en
* Editor elections
    * Elected 2
    * One starts now, replacing Gonzalo
    * One starts in August, replacing Tom
* Keeping retired editors involved
    * How do we engage with community members

**Comment by LukasBuecherl** [comment]
IWBDA/Biodesign Presentation:

- Highlight the applications of SBOL beyond just tutorials presented every year
- Demonstrate what is possible with SBOL in various applications
- Explain how both developers and biologists can utilize SBOL effectively
- Provide an example with SBOL2EXCL
- Prepare a 2-page abstract on the 10-year anniversary of SBOL Visual for Bio Design Week
- Present standardized build plans with an OpenTrons assembly demonstration

COMBINE:

- Submit the Visual paper, ensuring to meet the deadline (end of combine), preferably earlier
- Continue working on the Converter
